salt/journal/config/etc-nginx-sites-enabled-journal.conf

```diff
@@ -1,3 +1,1362 @@
+map_hash_bucket_size 518;
+
+map $uri $new_uri {
```

Comment:
if possible, it can be worth it to provide the map as a separate `file.managed` and `include` it; so that when we open up the nginx configuration it's not difficult to find relevant configuration.